src/coreclr/src/tools/aot/ILCompiler.ReadyToRun/Compiler/ReadyToRunCodegenCompilation.cs

```diff
@@ -484,15 +484,18 @@ protected override void ComputeDependencyNodeDependencies(List<DependencyNodeCor
                     catch (TypeSystemException ex)
                     {
                         // If compilation fails, don't emit code for this method. It will be Jitted at runtime
-                        Logger.Writer.WriteLine($"Warning: Method `{method}` was not compiled because: {ex.Message}");
+                        if (Logger.IsVerbose)
+                            Logger.Writer.WriteLine($"Warning: Method `{method}` was not compiled because: {ex.Message}");
```

Comment:
would it make sense to add `LogVerbose` method here which checks verbosity and logs?

Reply:
Yes, but I think we should think on a good api here. I'd rather do this, and then fix later.

Reply:
sounds good.